RELEASE/scripts/autoscend/iotms/mr2025.ash

```diff
@@ -0,0 +1,53 @@
+# This is meant for items that have a date of 2025
+
+boolean auto_haveMcHugeLargeSkis()
+{
+	if(auto_is_valid($item[McHugeLarge duffel bag]) && available_amount($item[McHugeLarge duffel bag]) > 0 )
+	{
+		return true;
+	}
+	return false;
+}
+
+boolean auto_equipAllMcHugeLarge()
+{
+	if (!auto_haveMcHugeLargeSkis())
+	{
+		return false;
+	}
+	if (!possessEquipment($item[McHugeLarge right pole]))
+	{
+		auto_openMcLargeHugeSkis();
+	}
+	autoForceEquip($slot[back]    , $item[McHugeLarge duffel bag]);
+	autoForceEquip($slot[weapon]  , $item[McHugeLarge right pole]);
+	autoForceEquip($slot[off-hand], $item[McHugeLarge left pole]);
+	autoForceEquip($slot[acc1]    , $item[McHugeLarge left ski]);
+	autoForceEquip($slot[acc2]    , $item[McHugeLarge right ski]);
+	return true;
+}
+
+boolean auto_openMcLargeHugeSkis()
+{
+	if (!auto_haveMcHugeLargeSkis())
+	{
+		return false;
+	}
+	if(possessEquipment($item[McHugeLarge right pole]))
+	{
+		return true;
+	}
+	//~ use($item[McHugeLarge duffel bag]); // does not work - need Mafia CLI tool?
```

Comment:
you can't `/use` it either so I think `visit_url` is the right move here. (and I don't expect Mafia to support `use`)